gems/sorbet/test/hidden-method-finder/update_hidden_methods_exp.sh

```diff
@@ -7,9 +7,9 @@ base_dir="$(pwd)"
 cd ../../../..
 # we are now at the repo root
 
-versions=("ruby_2_6" "ruby_2_7")
+versions=("sorbet_ruby_2_6" "sorbet_ruby_2_7")
 
-if ! bazel test //gems/sorbet/test/hidden-method-finder -c opt "$@"; then
+if ! bazel test //gems/sorbet/test/hidden-method-finder --define compiler=false -c opt "$@"; then
```

Comment:
I don't think we need `compiler=false` here, right?

Reply:
Yep, good catch.